.github/config/out_of_tree_extensions.cmake

```
@@ -100,6 +100,11 @@ duckdb_extension_load(sqlite_scanner
         GIT_TAG 50b7870be099186f195bc72bac5e9e11247ee2f9
         )
 
+duckdb_extension_load(sqlsmith)
```

```suggestion
duckdb_extension_load(sqlsmith
```
I think like this it should work